packages/dev/query/src/cli.js

```diff
@@ -569,32 +601,64 @@ export function run(input: string[]) {
       asset_node_modules: 0,
       asset_source: 0,
     };
+
+    let bg_ext = {};
+
+    let bg_type = {
+      entries: 0,
+      shared: 0,
+      async: 0,
+      parallel: 0,
+    };
+
     for (let [, n] of bundleGraph._graph.nodes) {
-      if (n.type in bg) {
+      if (n.type == 'bundle_group') {
+        // TODO: Counting priority here seems to work instead of counting on BundleNodes.. Counting DependencyNode priorities themselves
+        // gives different counts as well.
+        let priority = _getBundlePriority(bundleGraph, n);
+        if (priority == Priority.lazy) {
+          bg_type.async++;
+        } else if (priority == Priority.parallel) {
```

Comment:
Should've clarified this before but BundleGroups can't be parallel, only entry or async. A bundleGroup is a collection of bundles. Connected to it there may be multiple other bundles it pulls in, like a shared bundle or a parallel (aka type change) bundle. 

So the consideration for parallel should be done in the if statement where you check for shared. That might be why your example had 0 parallel bundles.